pychargifysimple/context_manager.py

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
 from contextlib import contextmanager
 
-import mechanicalsoup
 from fake_useragent import UserAgent
 
+import mechanicalsoup
```

Comment:
You don't must add a line break between fake_useragent and mechanicalsoup import.

Reply:
IMHO and from [isort](https://github.com/timothycrosley/isort) point of view module import `import <module>` should be separate from sub-module import `from <module> import <sub-module>` by a line break, that's why this line break is added